client/web/web.go

```diff
@@ -206,6 +223,7 @@ type nodeData struct {
 	IsUnraid          bool
 	UnraidToken       string
 	IPNVersion        string
+	Flag              string // empty when not running in any flagged mode
```

Comment:
Maybe `ForceMode` or `DebugMode` or something?  Just `Flag` doesn't help understand what this is used for or what values to expect.